app/assets/javascripts/admin/addon/lib/sidebar/admin-sidebar.js

```diff
@@ -384,13 +384,12 @@ export default class AdminSidebarPanel extends BaseCustomSidebarPanel {
       );
     }
 
-    for (const [sectionName, additionalLinks] of Object.entries(
-      additionalAdminSidebarSectionLinks
-    )) {
+    this.adminNavManager.filteredNavMap.forEach((section) => {
```

Comment:
I don't really like that a getter in a component is setting state on a service... but refactoring this is a way bigger change than I want to attempt in this PR.

Reply:
Yes feel free to suggest another way / followup refactor in another PR, it is a weird situation for sure.

This mutation should only ever happen once, if you see the admin nav manager I worked with CvX on last week, once `filteredNavMap` it can't be mutated anymore.

But yeah if there's a better way to do all this initial mutation not inside `sections()` then great!